client/package.json

```diff
@@ -0,0 +1,82 @@
+{
+  "name": "react-webpack-rails-tutorial",
+  "version": "1.1.0",
+  "description": "Built using the react_on_rails generator.",
+  "main": "server.js",
+  "engines": {
+    "node": "5.0.0",
+    "npm": "3.5.0"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/shakacode/react-webpack-rails-tutorial.git"
+  },
+  "keywords": [
+    "react",
+    "tutorial",
+    "comment",
+    "example"
+  ],
+  "author": "justin808",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/shakacode/react-webpack-rails-tutorial/issues"
+  },
+  "homepage": "https://github.com/shakacode/react-webpack-rails-tutorial",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node server.js",
+    "build:client": "NODE_ENV=production webpack --config webpack.client.rails.config.js",
+    "build:server": "NODE_ENV=production webpack --config webpack.server.rails.config.js",
+    "build:dev:client": "webpack -w --config webpack.client.rails.config.js",
+    "build:dev:server": "webpack -w --config webpack.server.rails.config.js",
+    "lint": "npm run eslint && npm run jscs",
+    "eslint": "eslint --ext .js,.jsx .",
+    "jscs": "jscs --verbose ."
+  },
+  "dependencies": {
+    "babel": "^6.3.26",
+    "babel-cli": "^6.4.5",
+    "babel-core": "^6.4.5",
+    "babel-loader": "^6.2.1",
+    "babel-plugin-syntax-decorators": "^6.3.13",
+    "babel-polyfill": "^6.3.14",
+    "babel-preset-es2015": "^6.3.13",
+    "babel-preset-react": "^6.3.13",
+    "babel-preset-stage-0": "^6.3.13",
+    "babel-types": "^6.4.5",
+    "es5-shim": "^4.5.0",
+    "expose-loader": "^0.7.1",
+    "imports-loader": "^0.6.5",
+    "jquery": "^2.2.0",
+    "jquery-ujs": "^1.1.0-1",
+    "loader-utils": "^0.2.12",
+    "lodash": "^4.0.0",
+    "react": "^0.14.6",
+    "react-bootstrap": "^0.28.2",
+    "react-dom": "^0.14.6",
+    "react-on-rails": "2.3.0",
```

Comment:
@justin808 this has nothing to do with the generator automation, if it's wrong here, it's because you didn't update the package.json.tt file in the generators